aarapsi_robot_pack/RequestDataset params
bool success
